src/types/env-type.ts

```diff
@@ -8,6 +8,9 @@ const envConfigSchema = Type.Object({
   OPENAI_API_KEY: Type.String(),
   NFT_MINTER_PRIVATE_KEY: Type.String({ default: "" }),
   NFT_CONTRACT_ADDRESS: Type.String({ default: "" }),
+  PERMIT_FEE_RATE: Type.String({ default: "" }),
```

Comment:
Any reason to only default this one?

Reply:
Removed default value for all permit related env variables https://github.com/ubiquibot/conversation-rewards/pull/43/commits/9ae0e17e8f64be114fb80f5f07c058edc5739697 to make it consistent